Aquarius/Src/Aquarius/Core/Layer.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include "Aquarius/Core/Utility.h"
+
+#include <deque>
+#include <string>
+
+namespace Aquarius {
+
+	class Event;
+	class Layer;
+
+	using LayerStack = std::deque<uniquePtr<Layer>>;
```

Comment:
Should we call this LayerDeque instead in the essence of being 100% transparent lol

Reply:
Stack does sound better than Deque though lol

Reply:
I'm gonna leave it as Stack because we are only ever gonna push Layers onto one end, and pop them from that same end